### PR TITLE
Added the ability to use any HTTP/1.1 method in http_check

### PR DESCRIPTION
**Pull request by benethor**

### CODE REVIEW

**Comment by pnuckowski** [comment]
:+1:

**Comment by ch40s3k** [comment]
:+1:

File: alamo_worker/plugins/http_check.py [hunk 1]
```diff
@@ -64,9 +64,12 @@ def execute(self, check):
         with self.statsd.timer('http_check.execute'):
             fields = check['fields']
             check['target'] = fields['url']
+            http_method = fields.get('method', 'get').lower()
             logger.info(
-                'Processing check: id="{}", url="{}", headers="{}"'.format(
```
Comment (deejay1) [inline alamo_worker/plugins/http_check.py]: if you're touching the logger maybe you could fix it to use %s and arguments instead of format()?

Comment (benethor) [inline alamo_worker/plugins/http_check.py]: sure, done

**Comment by pnuckowski** [comment]
:+1:

**Comment by h00dy** [comment]
💝

**Pull request merged**